src/silo/database.cpp

```diff
@@ -656,6 +647,10 @@ void Database::initializeColumn(config::ColumnType column_type, const std::strin
 void Database::initializeColumns() {
    for (const auto& item : database_config.schema.metadata) {
       initializeColumn(item.getColumnType(), item.name);
+      if (item.lineage_index) {
+         ASSERT(columns.indexed_string_columns.contains(item.name));
+         columns.indexed_string_columns.at(item.name).generateLineageIndex(lineage_tree);
+      }
```

Comment:
Why don't we make this part of `initalizeColumn`, in the `case config::ColumnType::INDEXED_STRING`? Then we also don't need the `ASSERT`, since the referenced value is created there.

Reply:
Because we then would need to pass the extra information from the metadata, but I will give it a shot

Reply:
We can simply pass the whole metadata struct?

Reply:
Yes, I did that now :)